src/redux/AppActions.js

```diff
@@ -6,42 +6,27 @@ export const myCheckButtonAction = function fetchCheckBox(value, value2) {
   };
 };
 
-export const myXSlideAction = function fetchXSlide(
-  valueMin,
-  valueMax,
-  volumeDataName
-) {
+export const myXSlideAction = function fetchXSlide(valueMin, valueMax) {
   return {
     type: "XSLIDE_CHANGED",
     payload: valueMin,
```

Comment:
Ideally, move this out of redux.  Minimally, rename the payloads to meaningful variable names and don't prefix all of the actions with `my`.